src/components/app-view/app-view.jsx

```diff
@@ -10,7 +10,7 @@ const AppViewBlock = styled.div`
 const AppView = props => {
     const { children } = props;
 
-    return <AppViewBlock>{children}</AppViewBlock>;
+    return <AppViewBlock {...props}>{children}</AppViewBlock>;
```

Comment:
you don't need to pass extra props between sidebar and app-view - please let me know if we can revert these 2 spreads.